Andlet/Andlet/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,6 +1,7 @@
 {
   "images" : [
     {
+      "filename" : "download.jpeg",
```

Comment:
Could we changed the name of this image in order to mention the propuse of itself ?

Reply:
Of course!!